README.md

```diff
@@ -33,6 +33,15 @@ It is made possible by [htmlwidgets](http://www.htmlwidgets.org/), which provide
 It can be installed from the R console: 
 ```r
 install.packages('devtools')
+library('devtools')
+```
+
+#### Install _goodpractice_  
```

Comment:
ℹ️ contributors/developers information should go to CONTRIBUTING.md. README is for user of the package and they don't need _goodpractice_ to use our package

Reply:
I don't think so. devtools is used in the installation steps below.
This section was added in #54 and generally, package installation are not that much verbose.
For instance, they prefer
```r
// install.packages('devtools')
// library('devtools')
devtools::install_github("process-analytics/bpmn-visualization-R@v0.2.1")
```